apps/token-manager/app/src/components/AutoComplete/AutoComplete.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useRef, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Transition, animated } from 'react-spring'
+import { ButtonBase, TextInput, springs, theme, unselectable } from '@aragon/ui'
+import { useClickOutside, useOnBlur } from '../../hooks'
+import IconMagnifyingGlass from './IconMagnifyingGlass'
+
+const { accent, contentBackground, contentBorder, textPrimary } = theme
+const identity = x => x
+
+const AutoComplete = React.memo(
+  React.forwardRef(function AutoComplete(
+    {
+      itemButtonStyles,
+      items,
+      onSelect,
+      onChange,
+      renderItem,
+      required,
+      value,
+      wide,
+    },
+    ref
+  ) {
+    const [opened, setOpened] = useState(true)
```

Comment:
Is it opened by default?

Reply:
Yes, but it won't show anything if there are no items and there will only be items when the user starts searching.

Reply:
Thee main reason for this is to close (`!open`) the items box when a user selects and item or looses focus without setting a null search.